### ISSUE TITLE
Provide a way to dismiss multiple error toasts at once.

### ISSUE THREAD

**Issue by beedell-roke** [warpdotdev/Warp#9734]
### Pre-submit Checks

- [x] I have [searched Warp feature requests](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3AFEATURE) and there are no duplicates
- [x] I have [searched Warp docs](https://docs.warp.dev) and my feature is not there

### Describe the solution you'd like?

- I should be able to <kbd>Shift</kbd>-select from the first to last, to dismiss them.

- Checkboxes should exist alongside each, for those with capacitive HIDs.

### Is your feature request related to a problem? Please describe.

I am unable to dismiss more than one error toasts at once:

https://github.com/user-attachments/assets/828ba81d-67b4-49fc-977a-e96fe807be75

### Additional context

To reproduce this, do as [`issues/9735`](https://github.com/warpdotdev/warp/issues/9735#issue-4364707732) explains, until more than one error toast is rendered.

### Operating system (OS)

Linux

### How important is this feature to you?

1 (Not too important)

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by peicodes** [comment]
I think we need mocks to determine the ideal experience for what happens when there are too many errors. It may be something like stacked errors rather than a global dismiss.

**Comment by RokeJulianLockhart** [comment]
@peicodes, I'd imagine that a resizable sidebar, like the vertical tab bar, would work well:

<p></p><img width="1276" height="357" alt="Image" src="https://github.com/user-attachments/assets/0cfa7bfa-441b-41ca-9541-9cce2437e372" /><p></p>

Then, even a single error wouldn't cover the search bar (as it currently does), etcetera.

**Comment by captainsafia** [comment]
@eluzzi5 is gonna take a look at adjusting our UX for notifications here to handle multiple error toasts more nicely. We'll incorporate that design mockup into any specs/implementation moving forward.

**Comment by eluzzi5** [comment]
Hey @RokeJulianLockhart thanks for submitting this issue!

Here are design mocks for improving the experience of multiple toasts: 
https://www.figma.com/design/UEnrVjd9IQO81Qbe9eSza4/Dismiss-multiple-error-toasts?node-id=47-32348&t=lJFw4w2WJ3LVJ8Lz-1
https://www.loom.com/share/12775188e8fc4b8185b0bb2743e6be31

**Comment by vkodithala** [comment]
Hey @beedell-roke - sorry for the delay here! Since we now have mocks, I've marked this issue `ready-to-implement`. Feel free to put up a PR :)
